lib/auth/init.go

```diff
@@ -358,35 +357,31 @@ func Init(cfg InitConfig, opts ...ServerOption) (*Server, error) {
 				Namespace: apidefaults.Namespace,
 			},
 			Spec: types.CertAuthoritySpecV2{
-				ClusterName:  cfg.ClusterName.GetClusterName(),
-				Type:         types.UserCA,
-				SigningKeys:  [][]byte{priv},
-				SigningAlg:   sshutils.ParseSigningAlg(sigAlg),
-				CheckingKeys: [][]byte{pub},
-				TLSKeyPairs:  []types.TLSKeyPair{{Cert: certPEM, Key: keyPEM}},
+				ClusterName: cfg.ClusterName.GetClusterName(),
+				Type:        types.UserCA,
+				ActiveKeys: types.CAKeySet{
```

Comment:
Since we're setting only new fields here, this means a newly provisioned 7.0 cluster can't be downgraded to 6.x - only 6.x upgraded to 7.0 can be downgraded back? This may be fine, just making sure this is behavior you wanted. Same below for host CA initialization.

Reply:
That's a good point, I didn't think about that.
Luckily, `UpsertCertAuthority` ends up filling in the old fields during `MarshalCertAuthority` so they end up populated in the backend.